### PR TITLE
Dec 15:  Quarren

### PR DESCRIPTION
**Pull request by GeekRuthie**
Issue #47

### CODE REVIEW

**Review by bigpresh — APPROVED** [review]
Excellent article, and I'm really interested in Quarren too - it sounds like a solid start to a decent, well designed CMS in Perl, which is something I've wanted to find for a while.  Kudos!

File: pending/15-whats-a-quarren.pod [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+=item * I don't want to see features I'm not using in the admin UI
+
+=item * Code I can be proud to give back to the community I love
+
+=back
+
+=head3 The "Core" of Quarren
+
+The core of the system includes pages, users, and the plugin infrastructure to find and utilize plugins.
+Literally everthing else is a plugin. A default renderer and theme is provided, to "pass through" raw HTML
```
Comment (racke) [inline pending/15-whats-a-quarren.pod:71]: s/everthing/everything/

Comment (GeekRuthie) [reply pending/15-whats-a-quarren.pod]: Good catch, @racke !  I've made the change.

**Comment by cromedome** [comment]
I love this article! :+1:

**Comment by cromedome** [comment]
Merged, thanks!

**Pull request closed without merging**